app/lib/forms/your_role.rb

```diff
@@ -0,0 +1,25 @@
+module Forms
+  class YourRole < Base
+    QUESTION_NAME = :employment_role
+
+    attr_accessor QUESTION_NAME
+
+    validates QUESTION_NAME, presence: true
```

Comment:
Weighing up the lack of repetition against the searchability.  No major feelings each way, more an observation than anything else.

Reply:
Yeah, it's not perfect imo. I think I'll need to think it over this evening.

Reply:
I think I'll stick with this way for now. I'm curious about the possibility of moving to sharing the attr_accessor/validation/permitted_params since they're repeated in multiple pages and this would be required for that.